config/splits/vpge/core.entity_form_display.paragraph.stanford_accordion.default.yml

```diff
@@ -0,0 +1,51 @@
+uuid: 3f4381f8-9c7b-4187-ad79-3448f83e0599
```

Comment:
This is defined already in the stack for both sites (not that diversityworks will use it anytime soon): https://github.com/SU-SWS/vpge_profile/blob/11.x/config/sync/core.entity_form_display.paragraph.stanford_accordion.default.yml

Is there an advantage to add it here too?

Reply:
This gets pulled into the config splits because there's an explicit split for `core.entity_form_display.*`.  Should be the same as in the sync directory.